src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <>
+      <h2>{`My name is ${name}`}</h2>
+      {age && (<p>{`I am ${age}`}</p>)}
+      <p>
+        {isMarried
+          ? (`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`)
```

Comment:
Move this to the variable
```suggestion
          ? (`${partnerName} is my ${....}`)
```